Content.Server/Medical/BiomassReclaimer/BiomassReclaimerSystem.cs

```diff
@@ -148,6 +150,19 @@ private void OnClimbedOn(EntityUid uid, BiomassReclaimerComponent component, Cli
             StartProcessing(args.Climber, component);
         }
 
+        private void OnRefreshParts(EntityUid uid, BiomassReclaimerComponent component, RefreshPartsEvent args)
+        {
+            var laserRating = args.PartRatings[component.MachinePartProcessingSpeed];
+            var manipRating = args.PartRatings[component.MachinePartYieldAmount];
+
+            //sloping down from 1/2 multiplier
+            component.ProcessingSpeedMultiplier =
+                component.BaseProcessingSpeedMultiplier * MathF.Pow(0.65f, laserRating - 1) + 0.1f;
+
+            //linear increase by .1 per rating
+            component.YieldPerUnitMass = component.BaseYieldPerUnitMass + (manipRating - 1) * 0.1f;
```

Comment:
The percentages should be data fields, rather than hard-coded.

Is there a reason why you picked different formulas for how the part ratings affect speed vs yield?